src/main/docker/Dockerfile

```diff
@@ -1,63 +1,70 @@
-FROM debian:stable-slim
-LABEL maintainer="steve.springett@owasp.org"
-LABEL vendor="OWASP"
+FROM eclipse-temurin:11.0.13_8-jre-focal AS jre-build
+
+FROM debian:bullseye-slim
 
 # Arguments that can be passed at build time
 # Directory names must end with / to avoid errors when ADDing and COPYing
+ARG COMMIT_SHA
+ARG APP_VERSION
 ARG APP_DIR=/opt/owasp/dependency-track/
 ARG DATA_DIR=/data/
-ARG USERNAME=dtrack
+ARG UID=1001
+ARG GID=1001
```

Comment:
Is there a specific reason why `1001` was chosen? The previous image used `1000`, so it's safe to assume that the permissions of most mounted volumes in current deployments are geared towards it (as in `1000:1000`). 

If we keep `1001` as the default, it should be documented in the upgrade notes. But as always, there will be quite a few users running into issues when blindly upgrading.

Reply:
ah my bad, I likely used the UID i used on another Image of mine I sourced for the changes. I will revert to `1000:1000`.
Thank you for that find!